languages/python/bitwarden_sdk/bitwarden_client.py

```diff
@@ -12,9 +12,10 @@ def __init__(self, settings: ClientSettings = None):
             settings_json = json.dumps(settings.to_dict())
             self.inner = bitwarden_py.BitwardenClient(settings_json)
 
-    def access_token_login(self, access_token: str):
+    def access_token_login(self, access_token: str,
+                           state_file_path: str = None) -> ResponseForAccessTokenLoginResponse:
```

Comment:
This function shouldn't be returning any type
```suggestion
                           state_file_path: str = None):
```

Reply:
Whoops